.github/workflows/publish.yml

```diff
@@ -4,9 +4,7 @@
 name: Create Release
 
 on:
-  # push:
-  #   tags:
-  #     - v*
+  workflow_dispatch:
```

Comment:
since there is nothing to follow, shouldn't there just be one line?: 
`on: workflow_dispatch`